### ISSUE TITLE
Panel with a hierarchical overview

### ISSUE THREAD

**Issue by geoffbaldwin** [SynBioDex/SBOLCanvas#196]
When there are multiple levels in a design it gets difficult/confusing knowing where you are in the design hierarchy. It would be really great to have a panel with a hierarchical overview, so you can see where you are.
It would also be great to be able to add a level above where you currently are, so for instance the TU you have just defined could become a sub-component of a larger design.

**Comment by cjmyers** [comment]
Partially handled with the new hierarchy views in lower right corner

**Comment by cjmyers** [comment]
Not sure if we are doing exactly what he wants here.  Let's discuss.

**Comment by geoffbaldwin** [comment]
Yes, the new panel in the lower right hand corner definitely helps. It would be good to be able to click on these to move between levels - seems like the natural thing to want to do.

**Comment by cjmyers** [comment]
Should be able to do that.  Is it not working?

**Comment by geoffbaldwin** [comment]
Yes, I noticed that after commenting, but it is really sporadic, sometimes clicking works but sometimes I keep clicking and I just can't get it to work and have to resort to the layer buttons.
